tests/metrics/classification/test_iou.py

```diff
@@ -0,0 +1,214 @@
+from functools import partial
+
+import numpy as np
+import pytest
+import torch
+from pytorch_lightning.metrics.classification.iou import IoU
+from pytorch_lightning.metrics.functional.iou import iou
+from sklearn.metrics import jaccard_score as sk_jaccard_score
+from tests.metrics.classification.inputs import (
+    _binary_inputs,
+    _binary_prob_inputs,
+    _multiclass_inputs,
+    _multiclass_prob_inputs,
+    _multidim_multiclass_inputs,
+    _multidim_multiclass_prob_inputs,
+    _multilabel_inputs,
+    _multilabel_prob_inputs
+)
+from tests.metrics.utils import NUM_CLASSES, THRESHOLD, MetricTester
+
+
+def _binary_prob_sk_metric(preds, target, average=None):
```

Comment:
can we rather reverse the names to be `_<lib>_<metric>` so to be `_sk_metric_binary_prob`
as we have in other tests (at least we shall have :)